src/test/java/homework_2/exercise2/Exercise2.java

```diff
@@ -0,0 +1,156 @@
+        softAssert.assertEquals(loginedUserName.getText(), "PITER CHAILOVSKII");
+
+        //5. Drop down "Service" in the header contains subcategories with options
+        driver.findElement(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']//*" +
+                "[@class='dropdown-toggle']")).click();
+
+        for (String option : listDropMenuService) {
+            WebElement element = driver.findElement(By.xpath(
+                    "//ul[@class='uui-navigation nav navbar-nav m-l8']//*" +
+                            "[@class='dropdown-menu']" +
+                            "//*[contains(text(),'" + option + "')]"));
+            softAssert.assertTrue(element.isDisplayed());
+        }
+
+        //6. Drop down "Service" in the left section contains subcategories with options
+        driver.findElement(By.xpath("//ul[@class='sidebar-menu']/li[@class='menu-title']/a")).click();
+        for (String option : listDropMenuService) {
+            WebElement element = driver.findElement(By.xpath(
+                    "//ul[@class='sidebar-menu']/li[@class='menu-title']" +
+                            "//*[contains(text(),'" + option + "')]"));
+            softAssert.assertTrue(element.isDisplayed());
+        }
+        // 7. Open through the header menu Service -> Different Elements Page
+        driver.findElement(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']//*" +
+                "[@class='dropdown-toggle']")).click();
+        driver.findElement(By.xpath(
+                "//ul[@class='uui-navigation nav navbar-nav m-l8']//*" +
+                        "[@class='dropdown-menu']" +
+                        "//*[contains(text(),'Different elements')]")).click();
+        softAssert.assertEquals(driver.getTitle(), "Different Elements");
+
+        //8.Check interface on Different elements page, it contains all needed elements: 4 checkboxes,
+        // 4 radios, 1 dropdown, 2 buttons
+        //check that there exactly 4 element for each
+        WebElement elements = driver.findElement(By.xpath("//div[@class='main-content-hg']/div[2]"));
+        List<WebElement> listOfElements = elements.findElements(By.tagName("label"));
+        softAssert.assertEquals(listOfElements.size(), 4);
+
+        elements = driver.findElement(By.xpath("//div[@class='main-content-hg']/div[3]"));
+        listOfElements = elements.findElements(By.tagName("label"));
+        softAssert.assertEquals(listOfElements.size(), 4);
+        //softAssert.assertEquals(countRadio, 4);
+        //dropdown element
+        elements = driver.findElement(By.xpath("//div[@class='main-content-hg']/div[@class='colors']"));
+        listOfElements = elements.findElements(By.tagName("select"));
+        softAssert.assertEquals(listOfElements.size(), 1);
+
+        //first buttons
+        softAssert.assertTrue(driver.findElement(By.xpath(
+                "//div[@class='main-content-hg']//*[@value='Default Button']")).isDisplayed());
```

Comment:
Please do not use `//*[]` as xpath for the element

Reply:
fix